CoEpi/ui/alerts/AlertsViewController.swift

```diff
@@ -7,26 +7,70 @@
 //
 
 import UIKit
+import RxSwift
+import RxCocoa
 
 class AlertsViewController: UIViewController {
+    private let viewModel: AlertsViewModel = .init()
+    private let dataSource: AlertsDataSource = .init()
 
+    @IBOutlet weak var titleLabel: UILabel!
     @IBOutlet weak var contactAlerts: UITableView!
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
+    private let disposeBag = DisposeBag()
 
-        // Do any additional setup after loading the view.
+    private static func formatTitleLabel(count: Int) -> String {
+        let title: String = "\(count) new contact alert"
+        
+        if count < 2 {
+            return title
+        }
+        return title + "s"
```

Comment:
Move this to the view model. The view controller should observe only a `Driver` with the text.